lerna.json

```diff
@@ -10,10 +10,6 @@
     },
     "version": {
       "message": "chore(release): publish"
-    },
-    "bootstrap": {
-      "ignore": "",
-      "npmClientArgs": ["--no-package-lock"]
```

Comment:
you can leave this in I think - just remove the `--no-package-lock`
in any case CLI params will take precedence so we can pipe additional custom stuff in from pipeline or elsewhere